exporters/otlp/otlptrace/example_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This example needs to be moved to the otlptracehttp module. The `go.opentelemetry.io/otel/exporters/otlp/otlptrace` package should not depend on `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp`.